src/utilities/customNumberFormat.ts

```diff
@@ -0,0 +1,12 @@
+
+    /**
+     *  Replace '.'  with ','  
+     *  Add space between thousands.
+     *  Optional: round number.
+     * @param value the value to be formatted. 
+     * @param decimals number of decimals. 
+     */
+    export function numberFormat(value: number, decimals?: number | undefined) {
+        const fractionDigits = decimals ? { minimumFractionDigits: decimals, maximumFractionDigits: decimals } : undefined;
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5707004)